src/abstractdataframe/abstractdataframe.jl

```diff
@@ -438,8 +438,7 @@ completecases(df)
 ```
 
 """
-function completecases(df::AbstractDataFrame)
-    ## Returns a Vector{Bool} of indexes of complete cases (rows with no NA's).
+function completecases(df::AbstractDataFrame)::AbstractVector{Bool}
```

Comment:
I like having the return type as part of the function definition cause then it's also self documenting and I didn't notice much of a difference benchmarking the two choices. That said, annotating `res` directly might result in more consistent performance in situations I didn't test against.